environments/production/app-processes.yml

```diff
@@ -89,6 +89,7 @@ celery_processes:
       concurrency: 6
       max_tasks_per_child: 5
 pillows:
+  pillow0: {}
```

Comment:
Isn't the rackspace one one supposed to be `{}`'d?

Reply:
I think that will be done in a separate step. Looks like the machine is just being added here, but no work has been assigned to it yet.

Reply:
Yeah that's right. I want to merge this so it gets fab deploy'd this afternoon, and then afterwards I can move the pillows over to this new machine in a separate changeset